### ISSUE TITLE
Demo app does not use local module

### ISSUE THREAD

**Issue by k-rister** [distributed-system-analysis/jschart#11]
When I use yarn to install the dependencies and launch the development environment it places a copy of the jschart module in the node_modules directory.  Then when local changes are made to the jschart module they are no reflected in the development environment.

I worked around this by creating a symbolic link form the node_modules/jschart directory to jschart directory but ideally this (or something similar) would happen automatically.

**Comment by gurbirkalsi** [comment]
@k-rister Does using this `npm link` command reflect changes in the testing environment automatically? 

```
(base) dhcp-10-20-1-63:tests guri$ npm link ../jschart
npm WARN jschart@1.0.6 No repository field.

audited 5347 packages in 2.205s
found 0 vulnerabilities

/Users/guri/.npm-packages/lib/node_modules/jschart -> /Users/guri/perfdev/jschart_test/jschart
/Users/guri/perfdev/jschart_test/tests/node_modules/jschart -> /Users/guri/.npm-packages/lib/node_modules/jschart -> /Users/guri/perfdev/jschart_test/jschart
```

**Comment by k-rister** [comment]
@gurbirkalsi When I run "npm link ../jschart" it looks like it is trying to modify the system level node modules (/usr/local/lib/node_modules) and gets an EACCES permission denied error and suggests running as root.  That seems like a less than ideal solution.

**Comment by k-rister** [comment]
@gurbirkalsi Turns out it also installed a copy in my ~/.npm directory that was overriding what was in node_modules -- so yet another static copy that wasn't reflecting the changes I was making but not in a different place.

**Comment by gurbirkalsi** [comment]
Can you try using `yarn link` instead? You do not need to be root to setup linking this way.

**Comment by k-rister** [comment]
That seems to work.  On a clean repository I did the following:

```
$ cd jschart
$ cd jschart
$ yarn link
$ cd ../tests
$ yarn link "jschart"
```

For verification I tested that changes were used and also this:

```
$ readlink -e node_modules/jschart
/home/krister/data/code/jschart/jschart
```

**Comment by gurbirkalsi** [comment]
Resolved with PR #12
